<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btn_home">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>HOME</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_table">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TABLES</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_sobre">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>SOBRE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_contato">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CONTATO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_outro">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>OUTRO</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="Pages">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pg_home">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(194, 194, 194);</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:48pt; font-weight:600; color:#7c3aed;&quot;&gt;DASHFY&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:600; color:#7c3aed;&quot;&gt;Veja. Controle. Decida.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pg_contato">
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>20</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Tela de Contato</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_sobre">
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>20</y>
          <width>121</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>SOBRE</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pg_cadastro">
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <widget class="QLabel" name="label_25">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Tela de Cadastro</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>CADASTRAR USUÁRIO</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Nome:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_nome"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Usuário:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_usuario"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_senha">
            <property name="text">
             <string>Senha:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_senha">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Confirmar senha:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_senha2">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Perfil:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_perfil">
            <item>
             <property name="text">
              <string>Usuário</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Administrador</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_19">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_cadastrar">
            <property name="text">
             <string>Cadastrar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pg_table">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tables">
           <attribute name="title">
            <string>TreeWidget</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLineEdit" name="txt_file"/>
              </item>
              <item>
               <widget class="QPushButton" name="btn_abrir">
                <property name="text">
                 <string>Abrir</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>ESTOQUE</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QTreeWidget" name="tw_estoque">
                    <column>
                     <property name="text">
                      <string>Nfe</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Serie</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Cliente</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>UF</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Municipio</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Cod Item</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Quantidade</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Descrição</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>UN</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Especie</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Valor Nfe</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Data Importação</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Usuario</string>
                     </property>
                    </column>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>SAÍDA</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QTreeWidget" name="tw_saida">
                    <column>
                     <property name="text">
                      <string>1</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Sério</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Data Importa</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Data Importação</string>
                     </property>
                    </column>
                    <column>
                     <property name="text">
                      <string>Usuario</string>
                     </property>
                    </column>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QFrame" name="frame_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <widget class="QPushButton" name="btn_estorno">
                   <property name="font">
                    <font>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Estorno</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btn_gerar">
                   <property name="font">
                    <font>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Gerar saída</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btn_importar">
                   <property name="font">
                    <font>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Importar</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="verticalSpacer">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cadastro">
        <property name="text">
         <string>CADASTRAR USUÁRIO</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
